cmd/boots/main.go

```diff
@@ -286,6 +288,7 @@ func newCLI(cfg *config, fs *flag.FlagSet) *ffcli.Command {
 	fs.StringVar(&cfg.logLevel, "log-level", "info", "log level.")
 	fs.StringVar(&cfg.dhcpAddr, "dhcp-addr", conf.BOOTPBind, "IP and port to listen on for DHCP.")
 	fs.StringVar(&cfg.syslogAddr, "syslog-addr", conf.SyslogBind, "IP and port to listen on for syslog messages.")
+	fs.StringVar(&cfg.kernelArgs, "kernel-args", "", "Kernel args to add to iPXE initrd for OSIE.")
```

Comment:
Can we do something like `extra-kernel-args, ..., "Extra set of kernel args that are appended to iPXE initrd for OSIE"` or something along those lines? I think that will avoid causing confusion for someone just looking around and running `-h`.

Reply:
updated.